js/ui/appDisplay.js

```diff
@@ -261,8 +261,18 @@ class BaseAppView {
 
     moveItem(item, newPosition) {
         let visibleItems = this._allItems.filter(item => item.actor.visible);
+
+        // Avoid overflow, this happens when trying to move an icon behind the
+        // AppCenterIcon
```

Comment:
This comment can be reduced to simply `Avoid overflow`

Reply:
done